inputs/templates/test/EvidenceQC/EvidenceQC.json.tmpl

```diff
@@ -12,5 +12,6 @@
   "EvidenceQC.manta_vcfs": {{ test_batch.manta_vcfs | tojson }},
   "EvidenceQC.melt_vcfs": {{ test_batch.melt_vcfs | tojson }},
```

Comment:
From here on out we will be running with scramble instead of MELT, so let's delete this as well
```suggestion
```

Reply:
Good point, thank you! I removed it from both files.